### PR TITLE
Create isabella-stewart-gardner-museum.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/comment/281298/#Comment_281298

[ISG Museum House Style.pdf](https://github.com/citation-style-language/styles/files/1177458/ISG.Museum.House.Style.pdf)
[Crivelli  Article entry 1_catalogue_final.pdf](https://github.com/citation-style-language/styles/files/1177459/Crivelli.entry.1_catalogue_final.pdf)
[Crivelli bibliography _catalogue_bib.pdf](https://github.com/citation-style-language/styles/files/1209286/Crivelli_catalogue_bib.pdf)

Very similar, Metropolitan Museum, shows the bibliography section: [Met Museum House Style.pdf](https://github.com/citation-style-language/styles/files/1177460/Met.Museum.House.Style.pdf)

The one question is why does it not print the bracket ) at the end of the publisher section? e.g.: Walter Isaacson. Steve Jobs. (New York, NY: Simon & Schuster, 2011.

### CODE REVIEW

**Comment by rmzelle** [comment]
> The one question is why does it not print the bracket ) at the end of the publisher section? e.g.: Walter Isaacson. Steve Jobs. (New York, NY: Simon & Schuster, 2011.

Are you asking why the guidelines are that way, or why the CSL style behaves that way?

**Comment by POBrien333** [comment]
Why the style behaves like that.

File: isabella-stewart-gardner-museum.csl [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+            <text variable="page"/>
+          </group>
+        </if>
+        <else-if type="book">
+          <text variable="title" prefix=" " suffix="." font-style="italic"/>
+          <group delimiter=", " prefix=" " suffix=".">
+            <text macro="volume-and-collection-title"/>
+            <text macro="edition"/>
+          </group>
+          <group delimiter=", " prefix=" (" suffix=")">
```
Comment (POBrien333) [inline isabella-stewart-gardner-museum.csl]: These closing brackets here do not appear for me :(
Looks like this: Walter Isaacson. Steve Jobs. (New York, NY: Simon & Schuster, 2011.
Should be: Walter Isaacson. Steve Jobs. (New York, NY: Simon & Schuster, 2011).

Comment (rmzelle) [inline isabella-stewart-gardner-museum.csl]: Is that in Mendeley? Visual Editor?

Comment (POBrien333) [inline isabella-stewart-gardner-museum.csl]: Visual Editor. I don't use Mendeley anymore. No use for it since I left academia ;)

Comment (rmzelle) [inline isabella-stewart-gardner-museum.csl]: I'm seeing this too in Zotero Standalone 4.0.29.17. If I remove the `display="block" it goes away. See the stripped down version at https://gist.github.com/rmzelle/e135ac9b7d6aaf6c6a2e5c99b3a55282.

Note that the specification says: "The display attribute (similar the “display” property in CSS) may be used to structure individual bibliographic entries into one or more text blocks. **If used, all rendering elements should be under the control of a display attribute.**"

Putting `display="block"` on the second group fixes the issue for me: https://gist.github.com/rmzelle/f54671436636b0e5d326c175f092f641

**Comment by rmzelle** [comment]
> Also, you did see the two other files I posted in this PR?

Yes, but I only briefly looked at them. Is there anything about them that I should be aware off in reviewing this CSL style?

(also, did you receive scans of the front matter that I requested of the requestor? I just want to make sure we name this style correctly, and that we have the bibliographic details on hand of the style guide in case we need to track it down in the future)

**Comment by POBrien333** [comment]
No, didn't get anything more sent via email. Let's wait and see.

**Comment by rmzelle** [comment]
@POBrien333, can you attach those additional files you received (especially the title page)? Also let me know if this should be good to merge.

**Comment by rmzelle** [comment]
(is this in good enough shape to merge right now? You can always submit a follow-up PR)

**Comment by POBrien333** [comment]
Let me send her an email. She never replied back on the forum. I think it still has some kinks to fix though.

**Comment by POBrien333** [comment]
Closing this, as I don't want to spend more time on this than I have already.

**Comment by adam3smith** [comment]
wouldn't it make sense to take it as is, though? It seems like it's pretty close.

**Comment by POBrien333** [comment]
if you want to, go ahead. :)

**Comment by rmzelle** [comment]
> I don't want to spend more time on this than I have already.

That's fine, of course. Did you tell the requestor?

**Comment by rmzelle** [comment]
And thanks!

**Pull request merged**